### ISSUE TITLE
[Bug]: Just looking for first GPU and not for second

### ISSUE THREAD

**Issue by Inazumanotenshi** [GPUOpen-LibrariesAndSDKs/Radeon-ReLive-VR#142]
**Describe the bug**
I have an ASUS ROG Zephyrus G13 Laptop. This one has a Ryzen 7 6800HS and a RX 6700S. I cant use ReLive cause its not shown in the AMD Software.

I read that this can be caused cause it needs 2 GB VRAM min to work. I sadly cant change the VRAM of the iGPU cause ASUS locks there bios to hard (40GB installed).
I could use it if I deactivate the iGPU and then reboot. However that sadly can not be a permanent way

**To Reproduce**
1. Have an active iGPU with 512mb VRAM and a dGPU with more then 2GB VRAM
2. Start AMD Adrenaline
3. Go to settings
4. (no ReLive VR shown)

**Setup (please complete the following information):**
 - OS: Windows 11 Pro
 - Headset Oculus Quest 1
 - Driver Version 24.3.1
 - ReLive version [cant see it]
 - GPU0 Radeon HD Graphics (Ryzen 7 6800HS)
 - GPU1 RX 6700S

**Expected behavior**
Maybe its possible to check for multiple GPUs and then say that it (if the iGPU has only 512mb) is forced to run on the dGPU.

**Comment by GennadiyAMD** [comment]
Do you have the c:\Program Files\AMD\WVR directory at all?

**Comment by Inazumanotenshi** [comment]
> Do you have the c:\Program Files\AMD\WVR directory at all?

yes, that directory exists and it contains a OpenVR folder with more files

**Comment by Inazumanotenshi** [comment]
Okay, after a CPU swap on my Desktop I can say it is caused by "looking for a wrong GPU0". Now my PC has a Ryzen 9 7900X3D and with 8GB VRAM for the iGPU it is not visible anymore. With my old CPU (Ryzen 5 5600X) it was showing, but not anymore. Anyone an Idea how to fix it without disable the iGPU?
